pom.xml

```diff
@@ -2078,11 +2078,15 @@
                 <groupId>ch.qos.logback.contrib</groupId>
                 <artifactId>logback-jackson</artifactId>
                 <version>0.1.5</version>
+                <scope>system</scope>
+                <systemPath>${project.basedir}/root/libs/logback-jackson-0.1.5.jar</systemPath>
```

Comment:
Remove to not mean add the linagora specific lib as a magic jar depedency.

BTW binary is not reviewable source code and do not meet ASF standards.